crates/locutus-stdlib/src/contract_interface.rs

```diff
@@ -36,6 +36,26 @@ pub enum ContractError {
     Other(String),
 }
 
+pub trait TryFromTsStd<T>: Sized {
+    fn try_decode(value: T) -> Result<Self, WsApiError>;
```

Comment:
must T be generic here or are we always decoding from a `&rmpv::Value`? Why does it need to be generic.

Reply:
In some implementations the type changes or is necessary to specify the lifetime